### PR TITLE
Bugfix/da 3300

### PR DESCRIPTION
**Pull request by lfpratik**

### CODE REVIEW

File: docker-images/base/patch/perceval-trigger-error-event.py.diff [hunk 1]
```diff
@@ -0,0 +1,435 @@
...
++
++
++def get_smtp_handler():
++    return SDSSMTPHandler(
++        mailhost=('smtp.gmail.com', 465),
++        fromaddr='dev.analytics.error.bot@gmail.com',
++        toaddrs=['pratikk@proximabiz.com', 'sgupta@linuxfoundation.org',
++                 'fghiasy@linuxfoundation.org'],
++        subject='SDS Error - log',
++        credentials=('dev.analytics.error.bot@gmail.com', 'Mobile@311!'),
```
Comment (lukaszgryglicki) [inline docker-images/base/patch/perceval-trigger-error-event.py.diff:273]: You know that SDS repo is public? A I can see password here.

File: sources/cmd/syncdatasources/syncdatasources.go [hunk 2]
```diff
@@ -39,6 +39,7 @@ var (
 	gHint             int
 	noDropPattern     = regexp.MustCompile(`^(.+-f-.+|.+-earned_media|.+-dads-.+|.+-slack|.+-da-ds-gha-.+|.+-social_media|.+-last-action-date-cache)$`)
 	notMissingPattern = regexp.MustCompile(`^(.+-github-pull_request.*|.+-github-issue-raw.*|.+-github-repository-raw.*)$`)
+	emailRegex        = regexp.MustCompile("^[a-zA-Z0-9.!#$%&'*+\\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$")
```
Comment (lukaszgryglicki) [inline sources/cmd/syncdatasources/syncdatasources.go:42]: Is this related to anyti8ng from this PR?

**Review by lukaszgryglicki — CHANGES_REQUESTED** [review]
Remove sensitive data from this PR.

File: docker-images/base/patch/perceval-trigger-error-event.py.diff (hunk 1 above)
Comment (lfpratik) [reply docker-images/base/patch/perceval-trigger-error-event.py.diff]: @lukaszgryglicki, Resolved

File: sources/cmd/syncdatasources/syncdatasources.go (hunk 2 above)
Comment (lfpratik) [reply sources/cmd/syncdatasources/syncdatasources.go:42]: I am not sure about this how it came

**Pull request closed without merging**